syne_tune/optimizer/schedulers/hyperband.py

```diff
@@ -540,7 +542,7 @@ def _on_config_suggest(self, config: Dict, trial_id: str, **kwargs) -> Dict:
             "bracket": kwargs["bracket"],
             "reported_result": None,
             "keep_case": False,
-            "running": True,
+            "status": SchedulerDecision.CONTINUE,
```

Comment:
I am a bit confused by this given that the doc says that the status of the trial is (running, paused, stopped) but what is stored is the scheduler decision. Could we unify the naming and the doc, for instance by using "scheduler_decision" rather than "status" given that we store a scheduler decision?